readthedocs/rtd_tests/fixtures/spec/v2/schema.json

```diff
@@ -0,0 +1,291 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$comment": "https://docs.readthedocs.io/en/stable/config-file/index.html",
+  "title": "Read the Docs",
+  "description": "Read the Docs configuration file.",
+  "type": "object",
+  "properties": {
+    "version": {
+      "title": "Version",
+      "description": "The version of the spec to use.",
+      "type": "string",
+      "enum": [
+        "2"
+      ],
+      "default": "2"
```

Comment:
This should be required, default is 1, which has a different scheme.

Reply:
It is marked as required in line 289! In a JSON Schema, we mark a property as required in the outer scope. (Unless I have it wrong.)

Reply:
Just to confirm, I tried with an arbitrary browser based JSON Schema validator.

![image](https://user-images.githubusercontent.com/22801583/123998666-febb1d80-d9d1-11eb-9b5f-7646ffa6ee37.png)

Reply:
oh, I see. We should remove the default value here, then.

Reply:
as the default is 1, and now it's just actually required for this scheme.

Reply:
Sure, sorry about that! Removed the the default value now.